File: sandbox/java/src/main/java/com/dropbox/sign_sandbox/OauthTokenGenerateDefaultExample.java
package com.dropbox.sign_sandbox;

import com.dropbox.sign.ApiException;
import com.dropbox.sign.Configuration;
import com.dropbox.sign.api.*;
import com.dropbox.sign.auth.*;
import com.dropbox.sign.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class OauthTokenGenerateDefaultExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        var oauthTokenGenerateRequest = new OAuthTokenGenerateRequest();
        oauthTokenGenerateRequest.clientId("cc91c61d00f8bb2ece1428035716b");
        oauthTokenGenerateRequest.clientSecret("1d14434088507ffa390e6f5528465");
        oauthTokenGenerateRequest.code("1b0d28d90c86c141");
        oauthTokenGenerateRequest.state("900e06e2");
        oauthTokenGenerateRequest.grantType("authorization_code");

        try
        {
            var response = new OAuthApi(config).oauthTokenGenerate(
                oauthTokenGenerateRequest
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling Oauth#oauthTokenGenerate");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: sandbox/python/src/OauthTokenGenerateDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    o_auth_token_generate_request = models.OAuthTokenGenerateRequest(
        client_id="cc91c61d00f8bb2ece1428035716b",
        client_secret="1d14434088507ffa390e6f5528465",
        code="1b0d28d90c86c141",
        state="900e06e2",
        grant_type="authorization_code",
    )

    try:
        response = api.OAuthApi(api_client).oauth_token_generate(
            o_auth_token_generate_request=o_auth_token_generate_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Oauth#oauth_token_generate: %s\n" % e)
